dependencies.toml

```diff
@@ -239,18 +239,39 @@ module = "org.bouncycastle:bcutil-jdk15on"
 version.ref = "bouncycastle"
 relocations = { from = "org.bouncycastle", to = "com.linecorp.armeria.internal.shaded.bouncycastle" }
 
-[libraries.brave]
+[libraries.brave5]
 module = "io.zipkin.brave:brave"
+version.ref = "brave5"
 # ":site:javadoc" fails when we use a newer version of Javadoc.
 javadocs = [
     "https://www.javadoc.io/doc/io.zipkin.brave/brave/5.6.3/",
     "https://www.javadoc.io/doc/io.zipkin.brave/brave-instrumentation-http/5.6.3/"]
-[libraries.brave-context-slf4j]
+[libraries.brave5-context-slf4j]
 module = 'io.zipkin.brave:brave-context-slf4j'
-[libraries.brave-instrumentation-http]
+version.ref = "brave5"
+[libraries.brave5-instrumentation-http]
 module = "io.zipkin.brave:brave-instrumentation-http"
-[libraries.brave-instrumentation-http-tests]
+version.ref = "brave5"
+[libraries.brave5-instrumentation-http-tests]
 module = "io.zipkin.brave:brave-instrumentation-http-tests"
+version.ref = "brave5"
+
+[libraries.brave6]
+module = "io.zipkin.brave:brave"
+version.ref = "brave6"
+# ":site:javadoc" fails when we use a newer version of Javadoc.
+javadocs = [
+    "https://www.javadoc.io/doc/io.zipkin.brave/brave/5.6.3/",
```

Comment:
that's a problem because the javadoc is for brave5...

Reply:
Fixed.